app/v2/controllers/V2MovementsController.scala

```diff
@@ -328,7 +328,9 @@ class V2MovementsControllerImpl @Inject() (
     def bodyExists(bodyAndSize: BodyAndSize): EitherT[Future, PresentationError, BodyAndContentType] =
       acceptHeader match {
         case VersionedRouting.VERSION_2_ACCEPT_HEADER_VALUE_JSON if bodyAndSize.size > config.smallMessageSizeLimit =>
-          EitherT.leftT[Future, BodyAndContentType](PresentationError.notAcceptableError("Large messages cannot be returned as json"))
+          EitherT.leftT[Future, BodyAndContentType](
+            PresentationError.notAcceptableError(s"Message are greater then ${config.smallMessageSizeLimit} bytes and can not retrieve as json")
```

Comment:
"Messages larger than ${config.smallMessageSizeLimit} bytes cannot be retrieved in Json"

@fmcgrath Can I get some advice on the error message that should go here? We'll return an error if a trader tries to get a message in Json that is larger than the small message size limit (5MB).

Reply:
@drnaylor-oc - I agree with your suggestion but I would write Json in ALL CAPS.

Messages larger than ${config.smallMessageSizeLimit} bytes cannot be retrieved in JSON